src/OpenCensus.Collector.Dependencies/DependenciesCollectorOptions.cs

```diff
@@ -16,10 +16,30 @@
 
 namespace OpenCensus.Collector.Dependencies
 {
+    using System;
+    using OpenCensus.Trace;
+    using OpenCensus.Trace.Sampler;
+
     /// <summary>
     /// Options for dependencies collector.
     /// </summary>
     public class DependenciesCollectorOptions
     {
+        private static Func<Uri, ISampler> defaultSampler = (uri) => { return uri.ToString().Contains("zipkin.azurewebsites.net") ? Samplers.NeverSample : Samplers.AlwaysSample; };
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DependenciesCollectorOptions"/> class.
+        /// </summary>
+        /// <param name="sampler">Custom sampling function, if any</param>
+        public DependenciesCollectorOptions(Func<Uri, ISampler> sampler = null)
```

Comment:
perhaps this func can take an `HttpRequest` object as an argument so one define a sampler based on cookies, not the url.